templates/docs/examples/patterns/buttons/dark.html

```diff
@@ -0,0 +1,29 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Buttons / Default{% endblock %}
+
+{% block standalone_css %}patterns_buttons{% endblock %}
+
+{% block content %}
+<div class="p-strip--dark is-shallow" style="background: #111">
+    <p>
+        <button class="p-button is-dark">Default button</button>
+        <button class="p-button is-dark" disabled>Default button disabled</button>
+    </p>
+    <p>
+        <button class="p-button--base is-dark">Base button</button>
+        <button class="p-button--base is-dark" disabled>Base button disabled</button>
+    </p>
+    <p>
+        <button class="p-button--positive is-dark">Positive button</button>
+        <button class="p-button--positive is-dark" disabled>Positive button disabled</button>
+    </p>
+    <p>
+        <button class="p-button--negative is-dark">Negative button</button>
+        <button class="p-button--negative is-dark" disabled>Negative button disabled</button>
+    </p>
+    <p>
+        <button class="p-button--brand is-dark">Brand button</button>
```

Comment:
@lyubomir-popov I haven't added dark theme to brand button yet - how would we create it from unknown color?

Reply:
We should have two variables, you can set both to the same current value